third_party/xla_client/custom_cuda_ops/bernoulli.cc

```diff
@@ -227,10 +228,13 @@ void LinearSumAssignment(CUstream stream, void** buffers, const char* opaque,
   int64_t* col_idx = reinterpret_cast<int64_t*>(buffers[2]);
   // std::cout << input << "," << (void*)mask << "," << output << std::endl;
 
-  xla::ShapeProto shape;
-  if (!shape.ParseFromArray(opaque, opaque_len)) {
+  if (opaque_len != sizeof(xla::LSAInfo)) {
     return;
```

Comment:
Ideally we want to throw errors in `XlaCustomCallStatus`, but this is fine for now.